README.md

```diff
@@ -100,35 +89,28 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **DataBase for managing animal clinic**
+- **Animal Table with sample records**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](<replace-with-your-deployment-URL>)
+- [Not available](<replace-with-your-deployment-URL>)
```

Comment:
[OPTIONAL]
- [ ] I would kindly suggest you remove the  `Live Demo` heading in the readme file. Since you didn't deploy the project yet then having the `Live Demo` heading with the wrong link will make your readme file unprofessional.